### ISSUE TITLE
Ordinals denoted as numeral with letter suffixes are handled improperly even though they are in the dictionary

### ISSUE THREAD

**Issue by jrencz** [aotaduy/eslint-plugin-spellcheck#8]
Words like `2nd` `3rd` are in the dictionary but they are reported as error by the plugin rule. 
Dictionary only has 1-9 variants. 10th would be an error even though it's  correct ordinal.

Good enough solution:

```json
"skipIfMatch": [
  "(\\s|^)\\d+(st|nd|rd|th)(\\s|$)"
]
```

**Comment by aotaduy** [comment]
I think I'm separting the 2 from the nd in my nodeWords methods. Let me check it out.

**Comment by aotaduy** [comment]
Done correcting this one, I added a test case too
